apps/wakunode2/config.nim

```diff
@@ -533,6 +555,17 @@ proc readValue*(r: var EnvvarReader, value: var crypto.PrivateKey) {.raises: [Se
   except CatchableError:
     raise newException(SerializationError, getCurrentExceptionMsg())
 
+proc readValue*(r: var TomlReader, value: var ProtectedTopic) {.raises: [SerializationError].} =
+  try:
+    value = parseCmdArg(ProtectedTopic, r.readValue(string))
+  except CatchableError:
+    raise newException(SerializationError, getCurrentExceptionMsg())
```

Comment:
Maybe we could reduce some code:
```suggestion
proc readValue*(r: var TomlReader, value: var ProtectedTopic) {.raises: [ConfigurationError].} =
    value = parseCmdArg(ProtectedTopic, r.readValue(string))
```

Reply:
mm unsure how this works under the hood, so just mirroring what we currently have in other `readValue` function parsing other arguments.